bin/restore.sh

```diff
@@ -9,7 +9,12 @@ restore_file=${1}
 if /app/bin/django_wait_for_db.sh
 then
 
-    echo "Dropping existing DB."
+    echo "Dropping existing DB in..."
+    echo "5"
```

Comment:
If you want to stick with this approach, you'll need to add `sleep`s after each `echo` for this to work.

Reply:
Not using this option.